src/Calculator.js

```diff
@@ -0,0 +1,139 @@
+import React, { Component } from 'react';
+import OperationButton from './OperationButton';
+import './Calculator.css';
+
+class Calculator extends Component {
+  MAX_NUMBER = 1000;
+  DIGIT_LIST = Array.from({ length: 10 }, (_, index) => 9 - index);
```

Comment:
```suggestion
  DIGIT_LIST = [9,8,7,6,5,4,3,2,1]
```
그냥 배열로 하는게 더 명시적이지 않을까 합니다. 숫자 체계가 변하지는 않을테니?!